src/components/MarcusFEMentor2/FooterComponents.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+
+import { ContactButton, HeaderBarLinks, MyTeamLogo } from './IndexPageComponents'
+import {
+  FooterAddressStyled,
+  FooterCopyrightStyled,
+  FooterDiv,
+  FooterLeftLogoBg,
+  FooterLowerContentDiv,
+  FooterLowerLeftStyled,
+  FooterLowerRightStyled,
+  FooterSocialMediaDiv,
+  FooterSocialMediaIconStyled,
+  FooterUpperContentDiv,
+  FooterUpperTextStyled,
+  LowerFooterBg,
+  UpperFooterBg,
+} from './styles/Footers.styled'
+
+import fbIcon from '../../assets/MarcusFEMentor2/icon-facebook.svg'
+import pinsIcon from '../../assets/MarcusFEMentor2/icon-pinterest.svg'
+import twtrIcon from '../../assets/MarcusFEMentor2/icon-twitter.svg'
+import FooterLeftLogo from '../../assets/MarcusFEMentor2/bg-pattern-home-6-about-5.svg'
```

Comment:
this should be camelCase instead; dont think there is a naming convention for it but try n keep it consistent